Formula/juise.rb

```diff
@@ -24,12 +24,22 @@ class Juise < Formula
   depends_on "sqlite"
 
   def install
-    system "sh ./bin/setup.sh" if build.head?
+    system "sh", "./bin/setup.sh" if build.head?
+
+    # Prevent sandbox violation
+    inreplace "configure",
+      "SLAX_EXTDIR=\"`$SLAX_CONFIG --extdir | head -1`\"",
```

Comment:
Ok, I see. I think it might still be worth asking upstream if we can configure that `extdir` or the call somehow?